server/build.gradle

```diff
@@ -32,15 +32,15 @@ dependencies {
   implementation 'io.javalin:javalin:3.12.0'
 
   // Mongo DB Driver for Java
-  implementation 'org.mongodb:mongodb-driver:3.12.7'
+  implementation 'org.mongodb:mongo-java-driver:3.12.7'
 
   // Jackson, a JSON library for Java
-  implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.2'
-  // implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.0'
+  // implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.2'
+  implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.0'
 
   // MongoJack, MongoDB integration for Jackson
-  implementation 'org.mongojack:mongojack:2.10.0'
-  // implementation 'org.mongojack:mongojack:4.0.2'
+  // implementation 'org.mongojack:mongojack:2.10.0'
+  implementation 'org.mongojack:mongojack:4.0.2'
```

Comment:
This version of mongojack requires the MongoDB driver 4.x

We should use
```gradle
implementation 'org.mongodb:mongodb-driver-sync:4.1.1
```

Reply:
Ah, so we can't upgrade Jackson that far without also upgrading Mongo. I had totally missed that part.

We can't just simply upgrade to either of the 4.1.1 drivers (`-sync` or `-legacy`) as both then break things in a variety of ways. So it does seem that we want/need to upgrade to Mongo 4 first, and then sort out the Jackson upgrade.

Reply:
We are already using Mongo 4 at this point so I would go ahead and start using the new driver.